source/request-as-event-emitter.ts

```diff
@@ -104,31 +113,32 @@ export default (options, input?: TransformStream) => {
 								return [];
 							}
 
-							const value = target[name];
+							const value = (target as any)[name];
 							return is.function_(value) ? value.bind(target) : value;
 						}
 					});
 				}
 
 				const {statusCode} = response;
-				response.statusMessage = response.statusMessage || http.STATUS_CODES[statusCode];
-				response.url = currentUrl;
-				response.requestUrl = requestUrl;
-				response.retryCount = retryCount;
-				response.timings = timings;
-				response.redirectUrls = redirects;
-				response.request = {options};
-				response.isFromCache = response.fromCache || false;
-				delete response.fromCache;
-
-				const rawCookies = response.headers['set-cookie'];
+				const typedResponse = response as unknown as Response;
```

Comment:
Hmm... What does `as unknown as Response` mean?

Reply:
I guess I don't need the `as unknown`. But it basically allows us to set the custom properties on the object